sqlitecluster/SQLiteNode.cpp

```diff
@@ -2132,12 +2132,29 @@ void SQLiteNode::_queueSynchronize(const SQLiteNode* const node, SQLitePeer* pee
         // Figure out how much to send it
         uint64_t fromIndex = peerCommitCount + 1;
         uint64_t toIndex = targetCommit;
-        if (!sendAll)
+        uint64_t timeoutLimitUS = 0;
+        if (sendAll) {
+            SINFO("Sending all commits with synchronize message, from " << fromIndex << " to " << toIndex); 
+
+            // We set this for all commits because this only gets all commits in response to SUBSCRIBE, which is done synchronously, and blocks the commit thread.
+            // For asynchronous queries, there's nothing being blocked, so it doesn't much matter how long these take.
+            // This is really not the correct encapsulation for this, but we can improve that later.
+            timeoutLimitUS = 10'000'000;
```

Comment:
Since the cluster gives up on the leader node after 30s, should we make this higher and closer to 30s, say 25? 

If there's a reason for 10 and it's not an arbitrary number, let's add a comment